tutoraspects/templates/aspects/jobs/init/init-lms.sh

```diff
@@ -40,5 +40,10 @@ EOF
 {% endif %}
 
 {% for model in EVENT_SINK_CLICKHOUSE_MODELS %}
+{% if ASPECTS_ENABLE_PII %}
 (./manage.py lms waffle_flag --list | grep event_sink_clickhouse.{{model}}.enabled) || ./manage.py lms waffle_flag --create event_sink_clickhouse.{{model}}.enabled --everyone
+{% elif model not in EVENT_SINK_CLICKHOUSE_PII_MODELS %}
+(./manage.py lms waffle_flag --list | grep event_sink_clickhouse.{{model}}.enabled) || ./manage.py lms waffle_flag --create event_sink_clickhouse.{{model}}.enabled --everyone
```

Comment:
If I'm reading this correctly, this will force the flag to be enabled even if the operator has intentionally disabled it? I think that if the flag exists we might want to leave it as-is instead of unexpectedly turning on PII to the database.

Reply:
Are you sure? It seems like it's grepping for `.enabled`?

Reply:
Yes, the flag is called `event_sink_clickhouse.course_overviews.enabled` but can be disabled from the waffle flag settings. If it's confusing we can change the name of the flag in `openedx-event-sink-clickhouse`